src/plugins/intel_gpu/src/kernel_selector/kernels/pooling/pooling_kernel_gpu_ref.cpp

```diff
@@ -9,6 +9,8 @@ ParamsKey PoolingKernelGPURef::GetSupportedKey() const {
     ParamsKey k;
     k.EnableInputDataType(Datatype::F16);
     k.EnableInputDataType(Datatype::F32);
+    k.EnableInputDataType(Datatype::UINT8);
+    k.EnableInputDataType(Datatype::INT8);
```

Comment:
could you explain why we need to support int8 from pooling_ref? Reference kernel does not provide good performance. If fs_b_yx_fsv32 is the problem, could you check why this format is chosen?

Reply:
fs_b_yx_fsv32 format and int8 input type is required on 4 E2E tests, TF_frozen_model_inception_v3_w_sym_ch_a_asym_t_acc1_78_34_INT8, TF_frozen_model_mobilenet_v2_w_sym_ch_a_asym_t_acc1_71_99_INT8, TF_Mobile_Object_Labeler_v1, TF_AiyVisionClassifier_Plants.

I've updated supporting fs_b_yx_fsv32 format on Pooling Int8Ref kernel instead of supporting it on ref kernel.